AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/ClientBehaviorTests/AWSAuthConfirmSignInTaskTests.swift

```diff
@@ -599,39 +599,7 @@ class AuthenticationProviderConfirmSigninTests: BasePluginTest {
             XCTFail("Should not return error \(error)")
         }
     }
-    
```

Comment:
Resource not found is handled differently now.

Reply:
Specifically because you retry now? Do we have tests for that?

Reply:
test written above handles scenario where resourceNotFound is returned.